build.sbt

```diff
@@ -128,7 +128,8 @@ lazy val arrowCore =
                     chronicleMap,
                     fastUtil,
                     commonsLang,
-                    junitInterface
+                    junitInterface,
+                    netty classifier osDetectorClassifier.value,
```

Comment:
Oh nice! Does this fix the windows issue by chance?

Reply:
Not sure, it is something Fabian did to try to reduce the number of package version conflicts